<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cgDbConverterView</class>
 <widget class="QMainWindow" name="cgDbConverterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>165</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>165</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ð¡oncierge: DbConverter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLabel" name="lblTo">
      <property name="text">
       <string>To :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblFrom">
      <property name="text">
       <string>From :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="NewFileFormat">
      <property name="toolTip">
       <string extracomment="New file format"/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pbChooseNewFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string extracomment="Choose path to save the new database"/>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pbChooseSourceFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string extracomment="Choose a database that you want to convert"/>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="SourceFile">
      <property name="toolTip">
       <string extracomment="Path to source file"/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="NewFile">
      <property name="toolTip">
       <string extracomment="Path to new file"/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="SourceFileFormat">
      <property name="toolTip">
       <string extracomment="Source format file"/>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Convert">
      <property name="text">
       <string>Convert</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
